litellm/proxy/management_endpoints/key_management_endpoints.py

```diff
@@ -2593,10 +2598,13 @@ async def _list_key_helper(
     # Prepare response
     key_list: List[Union[str, UserAPIKeyAuth]] = []
     for key in keys:
+        key_dict = key.dict()
```

Comment:
should also add a unit test for list_key_helper to make sure it attaches the object permissions u expect